package.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "near-fastauth-wallet",
-  "version": "0.0.14",
+  "version": "0.0.21",
```

Comment:
Just wondering, instead of increment the version, is there a reason why it is 21?

Reply:
Ah, that was for testing with verdaccio, let me return it to the previous version.
Good catch.

Reply:
Fixed